Formula/a/aws-nuke.rb

```diff
@@ -20,29 +20,27 @@ class AwsNuke < Formula
   depends_on "go" => :build
 
   def install
-    build_xdst="github.com/ekristen/aws-nuke/v#{version.major}/cmd"
+    build_xdst="github.com/ekristen/aws-nuke/v#{version.major}/pkg/common"
     ldflags = %W[
       -s -w
-      -X #{build_xdst}.BuildVersion=#{version}
-      -X #{build_xdst}.BuildDate=#{time.strftime("%F")}
-      -X #{build_xdst}.BuildHash=#{Utils.git_head}
-      -X #{build_xdst}.BuildEnvironment=#{tap.user}
+      -X #{build_xdst}.SUMMARY=#{version}
     ]
     with_env(
       "CGO_ENABLED" => "0",
     ) do
       system "go", "build", *std_go_args(ldflags:)
     end
 
-    pkgshare.install "config"
+    pkgshare.install "pkg/config"
 
     generate_completions_from_executable(bin/"aws-nuke", "completion")
   end
 
   test do
     assert_match "InvalidClientTokenId", shell_output(
```

Comment:
```suggestion
  test do
    assert_match version.to_s, shell_output("#{bin}/aws-nuke --version")
    assert_match "InvalidClientTokenId", shell_output(
```
Let's make sure this doesn't regress.